schema/loopring/fn_decode_float_24.sql

```diff
@@ -0,0 +1,14 @@
+DROP FUNCTION loopring.fn_decode_float_24;
+
+CREATE OR REPLACE FUNCTION loopring.fn_decode_float_24(data bytea) RETURNS double precision AS $$
```

Comment:
I think this exists as `fn_decode_float`, already? Can we delete that and use this one the old block_processing?